graphql_compiler/scripts/generate_test_sql/__init__.py

```diff
@@ -15,6 +15,4 @@
     ]
     sql_generator_strings = []
     for generator in sql_generators:
-        sql_generator_strings.extend(generator())
-
-    print '\n'.join(sql_generator_strings)
+        print(generator())
```

Comment:
Two things:
- I think this doesn't do the right thing, since the generator returns a list, and we don't want to print a list. Instead, we need to print one-element-per-line from the list.
- This should be caught by a test. Let's test that the generated file stored in the repo is the same file that running  the generation script produces.